File: apis/elysium-chat-api/src/main/java/com/seventh_root/elysium/chat/exception/ChatChannelMessagePreProcessingException.java
package com.seventh_root.elysium.chat.exception;

public class ChatChannelMessagePreProcessingException extends ChatChannelMessageProcessingException {
    public ChatChannelMessagePreProcessingException() {
    }

    public ChatChannelMessagePreProcessingException(String s) {
        super(s);
    }

    public ChatChannelMessagePreProcessingException(String s, Throwable throwable) {
        super(s, throwable);
    }

    public ChatChannelMessagePreProcessingException(Throwable throwable) {
        super(throwable);
    }

    public ChatChannelMessagePreProcessingException(String s, Throwable throwable, boolean b, boolean b1) {
        super(s, throwable, b, b1);
    }
}

File: bukkit/elysium-chat-bukkit/src/main/kotlin/com/seventh_root/elysium/chat/bukkit/exception/ChatChannelMessagePreProcessingException.kt
package com.seventh_root.elysium.chat.bukkit.exception

import com.seventh_root.elysium.chat.bukkit.exception.ChatChannelMessageProcessingException

class ChatChannelMessagePreProcessingException: ChatChannelMessageProcessingException {
    constructor() {
    }

    constructor(s: String): super(s) {
    }

    constructor(s: String, throwable: Throwable): super(s, throwable) {
    }

    constructor(throwable: Throwable): super(throwable) {
    }

    constructor(s: String, throwable: Throwable, b: Boolean, b1: Boolean): super(s, throwable, b, b1) {
    }
}
